pilot/pkg/networking/core/v1alpha3/accesslog.go

```diff
@@ -47,7 +47,7 @@ const (
 		"%DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% \"%REQ(X-FORWARDED-FOR)%\" " +
 		"\"%REQ(USER-AGENT)%\" \"%REQ(X-REQUEST-ID)%\" \"%REQ(:AUTHORITY)%\" \"%UPSTREAM_HOST%\" " +
 		"%UPSTREAM_CLUSTER% %UPSTREAM_LOCAL_ADDRESS% %DOWNSTREAM_LOCAL_ADDRESS% " +
-		"%DOWNSTREAM_REMOTE_ADDRESS% %REQUESTED_SERVER_NAME% %ROUTE_NAME%\n"
+		"%DOWNSTREAM_REMOTE_ADDRESS% %REQUESTED_SERVER_NAME% %DOWNSTREAM_PEER_URI_SAN% %ROUTE_NAME%\n"
```

Comment:
Is the current `REQUESTED_SERVER_NAME` enough for telling the mTLS status or maybe `DOWNSTREAM_TLS_VERSION` is better as it is more straightforward and covers both TLS and mTLS?

`DOWNSTREAM_PEER_URI_SAN` is also fine to me as it gives more information that can be useful in general debugging.

Either way, please also remember to update the documentation of the default access log: https://istio.io/latest/docs/tasks/observability/logs/access-log/#default-access-log-format

Reply:
> REQUESTED_SERVER_NAME

That's a good point actually.. That makes this less compelling